specifications/api/swagger.yaml

```diff
@@ -3789,6 +3828,25 @@ paths:
         - *parametersAddressState
       responses: *responsesListAddresses
 
+  /wallets/{walletId}/keys/{index}:
+    post:
+      operationId: postAccountKey
+      tags: ["Keys"]
+      summary: Create
+      description: |
+        <p align="right">status: <strong>stable</strong></p>
+
+        Retrieve account public key from the wallet.
```

Comment:
Perhaps a note would be handy:

"Note: Only `Hard` indexes are supported by this endpoint."

(similar as it is on https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/getWalletKey)

Reply:
thanks, done